src/bindings/python/docs/examples/openvino/mymodule/myclass.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+class MyClass():
+
```

Comment:
Consider provide example of docstrings for class/methods and what format shall be used e.g.: epydoc, google, reST etc.

Reply:
Will do!